data/remnant/remnant ships.txt

```diff
@@ -1035,6 +1035,60 @@ ship "Puffin"
 
 
 
+ship "Smew"
+	sprite "ship/smew"
+	thumbnail "thumbnail/smew"
+	attributes
+		category "Fighter"
+		licenses
+			Remnant
+		"cost" 1641000
+		"shields" 1200
+		"hull" 2800
+		"cargo space" 90
+		"fuel capacity" 100
+		"ramscoop" 0.8
+		"required crew" 1
+		"bunks" 9
+		"mass" 60
+		"drag" 1.96
+		"heat dissipation" .65
+		"outfit space" 92
+		"weapon capacity" 10
+		"engine capacity" 42
+		"shield generation" 0.48
+		"shield energy" 0.32
+		"hull repair rate" 0.9
+		"hull energy" 0.58
+		"outfit scan power" 14
+		"outfit scan speed" 1
+		"tactical scan power" 28
+		"asteroid scan power" 50	
+		weapon
+			"blast radius" 24
+			"shield damage" 130
+			"hull damage" 175
+			"hit force" 315
+	outfits
+		"Millennium Cell"
+		"Crystal Capacitor"
+		"Quantum Key Stone"
+		"Thermoelectric Cooler"
+		
+		"Crucible-Class Steering"
+		"Crucible-Class Thruster"	
```

Comment:
```suggestion
		"Anvil-Class Engine"
```

As a stock general purpose configuration, the Anvil works quite well, being reasonable fast, reasonably agile,  but not great at either. It also situates it ideally for for the variety of uses suggested in the description. (also makes it a bit cheaper: 290k instead of 372k for propulsion systems.)